src/analysis/retail/evoker/augmentation/modules/guide/CoreRotation.tsx

```diff
@@ -28,7 +28,10 @@ export function CoreRotationSection({ modules, events, info }: GuideProps<typeof
         <SpellLink spell={SPELLS.LIVING_FLAME_CAST} /> to fill in the gaps.
         <br />
         Your main cooldown, <SpellLink spell={TALENTS_EVOKER.BREATH_OF_EONS_TALENT} />, should be
-        used alongside your other DPS players' major cooldowns, since it amplifies their damage.
+        used alongside your other DPS players' major cooldowns, since it amplifies their damage. As
+        Scalecommander, it should instead be used as often as possible where mobs will survive the
+        full duration.
```

Comment:
The Scalecommander part should be conditional based on whether or not the player is using those Hero Talents.
```suggestion
        used alongside your other DPS players' major cooldowns, since it amplifies their damage. {info.combatant.hasTalent(TALENTS_EVOKER.MASS_ERUPTION_TALENT) && (As
        Scalecommander, it should instead be used as often as possible where mobs will survive the
        full duration.)}
```